56.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+class Solution:
+    def merge(self, intervals: List[List[int]]) -> List[List[int]]:
+        intervals.sort(key=self.compare)        
```

Comment:
Two issues:

1. This modifies the original list, which is not good. Use `my_sorted_list = sorted(input_list)` instead. Also, check the documentation and learn the difference between `sorted(some_list)` and `some_list.sort()`.
2. In Python, we usually use a lambda expression to specify the comparator, especially when the sort logic is simple like this one. 
```python
sorted_intervals = sorted(intervals, key=lambda x: x[0])

Reply:
Formally, lambdas are called anonymous functions, which comes in handy when a function definition is short and doesn't really need a function name due to the function's simplicity and its limited scope (no need to be visible by anyone else). It is a support provided by the programming language to simplify coding for cases like this one (as you can see, your original `compare` function consists of only one line of code).